bug_change_status_page.php

```diff
@@ -386,6 +387,33 @@
 <?php
 	}
 
+	if( $t_allow_file_upload ) {
```

Comment:
After merging as is, we would have nearly the same code block four times (bug_change_status_page.php, bug_report_page.php, bug_update_page.php, bugnote_add_inc.php)
Maybe you can introduce some kind of `function print_bugnote_attach_zone( ... )`